yoga/Yoga.c

```diff
@@ -335,6 +345,19 @@ int32_t YGNodeGetInstanceCount(void) {
   return gNodeInstanceCount;
 }
 
+YGConfigRef YGConfigNew(void) {
+  const YGConfigRef config = gYGMalloc(sizeof(YGConfig));
+  YG_ASSERT(config, "Could not allocate memory for config");
+  gNodeInstanceCount++;
```

Comment:
remove gNodeInstanceCount++ as this is not allocating a node. I don't think we care about tracking these allocations right now.